shared-helpers/src/utilities/MessageContext.ts

```diff
@@ -0,0 +1,51 @@
+import React, { FunctionComponent, createContext, createElement, useState, useRef } from "react"
+import { CommonMessageProps } from "@bloom-housing/ui-seeds/src/blocks/shared/CommonMessage"
+
+// TODO: this should be exportable from seeds directly
```

Comment:
Can we link a ticket for this work?

Reply:
@emilyjablonski Good point! I just wrote one up here: https://github.com/bloom-housing/ui-seeds/issues/83